tests/hedgehog_options/Categorifier/C/Hedgehog/Options.hs

```diff
@@ -71,6 +72,9 @@ checkRNG propertyName opts@HedgehogOptions {..} property = do
         H.RunnerConfig
           { H.runnerWorkers = Nothing,
             H.runnerColor = Nothing,
+#if MIN_VERSION_hedgehog(1, 1, 1)
+            H.runnerSeed = Nothing,
+#endif
```

Comment:
This is why it won't parse -- Ormolu only allows CPP at the top level (formatting in the presence of CPP is hard). So, here I would suggest pulling `runnerConfig` into its own definition, with conditionalization for the two versions:
```haskell
runnerConfig :: H.RunnerConfig
#if MIN_VERSION_hedgehog(1, 1, 1)
runnerConfig = H.RunnerConfig { ..., H.runnerSeed = Nothing }
#else
runnerConfig = H.RunnerConfig { ... }
#endif
```